test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -101,6 +101,7 @@ def test_utility_packages_using_import(training):
             executable="python",
         ).stdout.strip()
         if package == "sagemaker":
+            version = version.splitlines()[-1]
```

Comment:
If you are contributing this as a part of the PR, could you please change this to something like
https://github.com/aws/deep-learning-containers/pull/3293/files#diff-b38ca2feaf53682fc202ce43ba88a2d521bbfe77bddbd48040ce3562d42227ddR98-R107
with a regular expression to search for the version?